test/Test/Relude/Extra/Validation/Property.hs

```diff
@@ -0,0 +1,170 @@
+{-
+Copyright:  (c) 2016 Stephen Diehl
+            (c) 2016-2018 Serokell
+            (c) 2018-2019 Kowainik
```

Comment:
I believe there's no need to specify everyone here, since it's a brand new module. So this can be just:

```haskell
Copyright:  (c) 2019 Kowainik
```